app/assets/javascripts/search.js

```diff
@@ -2,6 +2,7 @@
 //= require jquery
 //= require jquery_ujs
 //= require jquery.color-2.1.2
+//= require util/js.cookies
```

Comment:
js.cookies is a vendor lib and therefore should be in the ```vendor``` dir. (take a look at that directory, we have all the js dependencies stored there.)